From: Jiri Slaby <jslaby@suse.cz>
Subject: Remove timestamp

Nobody wants timestamps, it's bogus as it causes irreproducible
builds. Remove that crap from api_log.

---
 src/api/api_log.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: z3-4.8.3+git.20181121/src/api/api_log.cpp
===================================================================
--- z3-4.8.3+git.20181121.orig/src/api/api_log.cpp
+++ z3-4.8.3+git.20181121/src/api/api_log.cpp
@@ -49,7 +49,7 @@ extern "C" {
                 res = false;
             }
             else {
-                *g_z3_log << "V \"" << Z3_MAJOR_VERSION << "." << Z3_MINOR_VERSION << "." << Z3_BUILD_NUMBER << "." << Z3_REVISION_NUMBER << " " << __DATE__ << "\"\n";
+                *g_z3_log << "V \"" << Z3_MAJOR_VERSION << "." << Z3_MINOR_VERSION << "." << Z3_BUILD_NUMBER << "." << Z3_REVISION_NUMBER << "\"\n";
                 g_z3_log->flush();
                 g_z3_log_enabled = true;
             }
